assets/lang/sklmessages_pl_PL.properties

```diff
@@ -83,21 +83,21 @@ settings.title=Ustawienia launchera
 settings.lang=Język
 settings.lang.help=Widzisz błąd w tłumaczeniu?
 settings.lang.link=Pomóż go naprawić!
-settings.theme=Motyw
+settings.theme=Motywy
```

Comment:
Tutaj zmiana na lepsze. Chociaż tekst `settings.theme` każe sugerować, że powinna być tutaj liczba pojedyncza. I, podobnie jak wyżej, będzie się to chyba wiązało ze zmianą w pliku źródłowym.
A tak przy okazji, czy ten tekst jest w ogóle gdzieś używany?

Reply:
Z kontekstu wynika, że właśnie tam powinien się on znajdować, ale u mnie nigdzie go nie ma, nawet tam. Załączyłbym fragment zrzutu ekranu przedstawiający jak to u mnie wygląda, ale nie widzę w tym większego sensu. Musisz wierzyć mi na słowo. Rozchodzi się o to, że jeśli tekst ten nie jest używany, to niepotrzebnie "zmuszamy" tłumaczy nowych języków do jego tłumaczenia. Niby jeden tekst to niewiele, ale to zawsze coś.

Reply:
Rzeczywiście nie ma napisu "Motyw". Też mi się tak wydawało, że napis "Motyw" jest nad przyciskami "Jasny motyw" i "Ciemny motyw". :D

Reply:
Rzeczywiście go tam nie ma 😁 Więc chyba wszystko jedno, jak będzie przetłumaczone

Reply:
Była to pozostałość po poprzednim rozwiązaniu. Oryginalnie `settings.theme` był opisem rozwijanej listy z dostępnymi motywami.